### DISCUSSION TITLE
Any way to get SynoAI notification work in combination with own hosted NTFY Server?

### DISCUSSION THREAD

**Discussion by Ma7e1337** [djdd87/SynoAI#201] [Q&A]
Hi Guys,

is there any way to get SynoAI working in combination with an NTFY Server?
the other services like telegram, which are listed in the wiki are no option for me.

i hope you can help me and maybe you have an example for the notification-part in the appsettings,json?
anything else is working..

Thank you
best regards

**Comment by djdd87** [comment]
NTFY isn't something I've come across, but it just looks like a REST API. Have you tried to see if you can use the Webhook notification type with it?

**Reply by Ma7e1337** [reply]
Unfortunately, I don't have the best programming knowledge and especially not in relation to the individual programming languages, so I apologize if you ask a few "stupid questions" ;)


 The NTFY wiki says the following:

![Screenshot_20230928-111244_1.png](https://github.com/djdd87/SynoAI/assets/140547189/f04e4b9a-7e27-4ae8-b513-fdf2114449da)
(https://docs.ntfy.sh/publish/#webhooks-publish-via-get)

The Notifiers section of my config is as follows:

"Notifiers": [
 { 
"Type": "Webhook",
"Url": "http://myip/Alarm", 
"Method": "POST", 
"Field": "image", 
"SendImage": true, 
"AllowInsecureUrl": true

So it seems as if notifications work via NTFY via webhook in principle. However, I don't know what exactly I have to write in the config so that the notification is received on my NTFY server.

I would be very happy if I could be helped.  Best regards ;)

**Reply by JoeyJoe6** [reply]
That entry looks fine in your synoai appconfigs.conf file except your missing a } at the end }, if you have more Notifiers listed. 
Is that webhook not working? let us know
May have to modify the "url" to include values instead

**Reply by djdd87** [reply]
Unless the structure matches something similar to what I'm posting from a Webhook, it's not going to work as intended. Ideally we'd able to specify a message template for the POST body, which is something I'm doing in the next version #155
